dev/arktype.io/docs/api/operators.md

```diff
@@ -0,0 +1,22 @@
+---
+hide_table_of_contents: true
+---
+
+# Operators
```

Comment:
I really like this! I think it will be a huge help to people as a cheat sheet 👍

Let's actually have the operators themselves (i.e. "&" instead of "intersection") as the left column.

Also, maybe we just leave the ones that don't exist blank instead of having so many ❌ that make it look like our library is missing features KEKW